validator/api/src/main/java/tech/pegasys/teku/validator/api/ValidatorApiChannel.java

```diff
@@ -85,4 +86,6 @@
 
   SafeFuture<List<SubmitCommitteeSignatureError>> sendSyncCommitteeSignatures(
       List<SyncCommitteeSignature> syncCommitteeSignatures);
+
+  void sendContributionAndProofs(List<SignedContributionAndProof> signedContributionAndProofs);
```

Comment:
This will need to return a `SafeFuture` if we're going to report any failures back.  Probably makes sense to have it just return `SafeFuture<Void>` and if it fails the future completes exceptionally.